ops/model.py

```diff
@@ -1044,62 +1034,59 @@ class Container:
     Attributes:
         name: The name of the container from metadata.yaml (eg, 'postgres').
     """
-
     def __init__(self, name, backend, pebble_client=None):
         self.name = name
 
         if pebble_client is None:
             socket_path = '/charm/containers/{}/pebble.socket'.format(name)
             pebble_client = backend.get_pebble(socket_path)
         self._pebble = pebble_client
-        self._completed = None
 
     @property
     def pebble(self) -> 'pebble.Client':
         """The low-level :class:`ops.pebble.Client` instance for this container."""
         return self._pebble
 
-    @property
-    def completed(self) -> bool:
-        """Whether or not a :meth:`is_ready` context finished successfully."""
-        return self._completed
-
-    def is_ready(self) -> '_ContainerReady':
+    def can_connect(self) -> bool:
         """Check whether or not Pebble is ready as a simple property.
```

Comment:
This is no longer "a simple property", just a regular method. Also, I really like the Go documentation's use of "report" to denote which way around the boolean is (though in this case the method name is very clear). How about changing to "Report whether Pebble is ready."?

Reply:
Done, thanks! :)